modules/nf-core/spades/tests/main.nf.test

```diff
@@ -0,0 +1,198 @@
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out.scaffolds,
+                    process.out.contigs,
+                    process.out.transcripts,
+                    process.out.gene_clusters,
+                    process.out.gfa,
+                    process.out.versions
+                    ).match() },
+                { assert path(process.out.log[0][1]).readLines().any { it.contains("SPAdes pipeline finished") } },
+                { assert file(process.out.warnings[0][1]).find{ file(it).name == "warnings.log"} }
+            )
+        }
+    }
+
+    test("sarscov2 - pe - nanopore ") {
+
+        when {
+            process {
+                """
+                input[0] = [ [ id:'test', single_end:false ], // meta map
+                            [ file(params.modules_testdata_base_path + "genomics/sarscov2/illumina/fastq/test_1.fastq.gz", checkIfExists: true),
+                                file(params.modules_testdata_base_path + "genomics/sarscov2/illumina/fastq/test_2.fastq.gz", checkIfExists: true) ],
+                            [],
+                            [ file(params.modules_testdata_base_path + "genomics/sarscov2/nanopore/fastq/test.fastq.gz", checkIfExists: true) ]
+                        ]
+                input [1] = []
+                input [2] = []
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out.scaffolds,
+                    process.out.contigs,
+                    process.out.transcripts,
+                    process.out.gene_clusters,
+                    process.out.gfa,
+                    process.out.versions
+                    ).match() },
+                { assert path(process.out.log[0][1]).readLines().any { it.contains("SPAdes pipeline finished") } },
+                { assert file(process.out.warnings[0][1]).find{ file(it).name == "warnings.log"} }
+            )
+        }
+    }
+
+    test("sarscov2 - pe - nanopore - yml ") {
+        when {
+            process {
+                """
+                input[0] = [ [ id:'test', single_end:false ], // meta map
+                            [ file(params.modules_testdata_base_path + "genomics/sarscov2/illumina/fastq/test_1.fastq.gz", checkIfExists: true),
+                                file(params.modules_testdata_base_path + "genomics/sarscov2/illumina/fastq/test_2.fastq.gz", checkIfExists: true) ],
+                            [],
+                            [ file(params.modules_testdata_base_path + "genomics/sarscov2/nanopore/fastq/test.fastq.gz", checkIfExists: true) ]
+                        ]
+                input [1] = file(params.modules_testdata_base_path + "delete_me/spades/spades_input_yml.yml", checkIfExists: true)
+                input [2] = []
```

Comment:
I don't see any test that would use input[2] (`hmm`). Is it possible to add one?

Reply:
You are right, there should be one. I added hmm test.